src/mosaic-node-manager.cc

```diff
@@ -221,43 +284,33 @@ namespace ns3 {
             return;
         }
         NS_LOG_INFO("Mosaic MosaicNodeManager::SendMsg " << nodeId);
-
-        Ptr<Node> node = NodeList::GetNode(m_mosaic2ns3ID[nodeId]);
+        Ptr<Node> node = NodeList::GetNode(nodeId);
```

Comment:
Can you explain why we no longer need the ID lookup here? Have we changed the kinds of IDs were passing into this function?

Reply:
The change was incorrect. The 'GetNs3NodeId' function is called before the 'SendMsg' function to retrieve the correct ID. This means that the 'nodeId' passed to the 'SendMsg' function is the converted ID

Reply:
Updated this incorrect change